definitions/apm-application/relationships.yml

```diff
@@ -0,0 +1,37 @@
+relationships:
+  # Select entityGuid, peer.hostname from Span where peer.hostname is not null and category =
+  # 'datastore'
+  # ExtService in this case is a DB
+  #
+  # TODO revise rule 
+  - instrumentation.provider: newRelic
```

Comment:
For me this is an example of why it'd make sense to have these in the definitions file itself.  In cases like OTEL, the instrumentation.provider tag is not present in the original telemetry (we add a tag rule to copy it from telemetry.sdk.name) and there is a rule so I can imagine a user being confused by what to put here or where that value comes from.  Maybe worth checking with product to see which side of the tradeoff is better here.